Makefile

```diff
@@ -11,17 +11,17 @@ GOBIN         = $(shell go env GOPATH)/bin
 endif
 GOX           = $(GOBIN)/gox
 GOIMPORTS     = $(GOBIN)/goimports
+GOLANGCI_LINT = $(GOBIN)/golangci-lint
 ARCH          = $(shell go env GOARCH)
 
 ACCEPTANCE_DIR:=../acceptance-testing
-# To specify the subset of acceptance tests to run. '.' means all tests
```

Comment:
None of these comment changes should be part of this PR. Create another one about comments if you like, but I don't think that will go anywhere.